### ISSUE TITLE
ProMicro Slave Jumper Clarification

### ISSUE THREAD

**Issue by zachfedor** [nicinabox/lets-split-guide#12]
In the [assembly instructions](https://github.com/nicinabox/lets-split-guide/blob/master/assembly.md#one-side-isnt-working), you have a troubleshooting note about the bridged J1 jumper on the Pro Micros. I'm running into this issue, where the master side works fine but there is no signal from the slave.

I'm using SparkFun's Pro Micros and both have shorted J1s. Should I remove the solder bridge from one or both of the micros?

**Comment by nicinabox** [comment]
I believe so. I haven't tested it on a Sparkfun version, but I did test it both ways on the common clone version and it wouldn't work when jumpered. The schematic shows a closed J1 bypasses a bunch of stuff and puts it into 5V mode. I don't really know why 5V doesn't work though.

Worst case if you find theres no change, just put a dab of solder back on it.

![arduino-pro-micro-schematics](https://user-images.githubusercontent.com/246280/27453658-d5ba751c-575c-11e7-9fbb-fdee2a03df9b.jpg)

**Comment by zachfedor** [comment]
good point. i'll give it a shot tonight and update with my findings

**Comment by zachfedor** [comment]
No luck.

I removed the solder bridge from both Pro Micros and I'm still not receiving any input from the right hand side. Ongoing discussion is happening in [help a help thread I started on /r/olkb](https://www.reddit.com/r/olkb/comments/6hxawx/help_with_lets_split/). Someone else is having the same issue, too, and we're talking it out [here](https://www.reddit.com/r/olkb/comments/6ixnip/help_flashing_lets_split/).

**Comment by nicinabox** [comment]
@zachfedor any updates on this?

**Comment by zachfedor** [comment]
Kind of. When the left hand was master, I couldn't get any signal from the right hand. The ProMicros LEDs were constantly lit, which I believe is a good hint that something ain't right. When I plugged the right hand in as master, I could get key presses from the left hand as a slave. Obviously the default keymap was mirrored this way, but at least something was working.

I eventually gave up trying to find the issue. Instead, I modified my keymap to force right hand master and can only plug it in from that side now. I also had to change the function that determined which side had the USB plugged in, which seemed counter-intuitive. But it works!

Here is the commit containing those code changes: https://github.com/zachfedor/qmk_firmware/commit/42655a4870ad73ecbce4c663957c1cd3f4baa773
